dataconverters.py

```diff
@@ -89,17 +87,18 @@ def convertMMCToXarrayNCDF(pathbase,year,dataDir,ncDir):
     print("\n")
 
 def dbToXarray(db,specified_date=None):
-    """Convert db to xarray
+    """Convert db instance of MMCData class from mmcdata.py to xarray
 
     If specified_date is not None, then the Time array will use the
     specified date; otherwise, the date will be read from the input 
     database. This is used as a hack to handle LLNL's bogus dates and
     times since the WRF run was 'ideal'.
     """
     #Deal with the times by converting the metadata strings to datetime objects
-    Times = np.ndarray([len(db)-1])
-    for i in range(1,len(db)):
-        #date_string='{:s} {:s}'.format(db[i][0]['date'],db[i][0]['time']).strip()
+    print(type(db))
+    Times = np.ndarray(db.dataSetLength)
+    for i in range(1,db.dataSetLength):
```

Comment:
To clean up all of the nested `db[i][0][field]` quantities in this block, could you just ditch the `i` counter?
E.g., I think this is cleaner: 
```
for row in db[1:]:
    row = row[0]  # (I don't recall why/how this is stored this way)
    now_do_something_with(row['date'], row['time'])
    #etc 
```

Reply:
I've removed the routine entirely since it is simply an artifact now and shouldn't be used at all.